lib/.eslintrc.yaml

```diff
@@ -40,3 +40,5 @@ globals:
   DCHECK_LE: false
   DCHECK_LT: false
   DCHECK_NE: false
+  DLOG: false
+  internalBinding: false
```

Comment:
I would rather add a eslint comment in the file where `internalBinding` is needed.